component-samples/demo/reseller/WEB-INF/web.xml

```diff
@@ -81,10 +81,14 @@
         <role-name>api</role-name>
     </auth-constraint>
     <user-data-constraint>
-        <transport-guarantee>NONE</transport-guarantee>
+       <transport-guarantee>CONFIDENTIAL</transport-guarantee>
     </user-data-constraint>
   </security-constraint>
 
+  <login-config>
+       <auth-method>CLIENT-CERT</auth-method>
```

Comment:
Can't we configure it via `service.yml` file ? like `<auth-method>$(AUTH_METHOD)</auth-method>`

Reply:
we can change the web.inf to <auth-method>$(AUTH_METHOD)</auth-method> so it can be set via java env/property in service.yml.